apps/new-dealer/views/partials/weapons-ammo-unspecified.html

```diff
@@ -1,8 +1,6 @@
-<div id="{{toggle}}-panel" class="reveal" aria-hidden="false">
-  <div class='panel-indent'>
+<div id="{{toggle}}-panel" class="govuk-checkboxes__conditional govuk-checkboxes__conditional--hidden" aria-hidden="false">
     {{#textarea}}{{toggle}}{{/textarea}}
-  </div>
 </div>
-<div class="form-block">
+<div class="govuk-checkboxes__divider">
```

Comment:
what's a `govuk-checkboxes__divider`

Reply:
You can use this class to format where checkbox groups where one option is separate from the others e.g. when you need to convey an either/or scenario you would usually use the divider with the word ‘or’ .